formulas/operators.js

```diff
@@ -56,22 +56,26 @@ const Infix = {
   compareOp: (value1, infix, value2, isArray1, isArray2) => {
     if (value1 == null) value1 = 0;
     if (value2 == null) value2 = 0;
+    if(isArray1 || isArray2)
+      return FormulaHelpers.equalOP(value1, value2, infix)
+      
+
     // for array: {1,2,3}, get the first element to compare
     if (isArray1) {
       value1 = value1[0][0];
     }
     if (isArray2) {
       value2 = value2[0][0];
     }
-
+      
     const type1 = typeof value1,
       type2 = typeof value2;
 
     if (type1 === type2) {
       // same type comparison
-      switch (infix) {
+      switch (infix) {        
```

Comment:
Use the same map I suggested above, put it in a util if you have to and import in both locations